pkg/syncserver/sync_server.go

```diff
@@ -265,6 +290,22 @@ func (s *Server) serve(cxt context.Context) {
 			return
 		}
 
+		logCxt.Infof("Accepted from %s", conn.RemoteAddr())
+		tlsConn, ok := conn.(*tls.Conn)
```

Comment:
Will we also accept non-TLS connections?  I think we should probably drop them if we're in TLS mode.  WDYT?